cross/dotnet-runtime/Makefile

```diff
@@ -0,0 +1,42 @@
+PKG_NAME = aspnetcore-runtime
```

Comment:
Can we drop the `asp` and `core`? AFAIK the package is just called `.net` or `dotnet` with just 2 variations either SDK or runtime

I thought the `asp` part is the proprietary windows APIs

Reply:
not it isn't  (see https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
There are three variants of runtimes to download
-  `ASP.NET Core Runtime` including `.NET Runtime`
-  `.NET Desktop Runtime`  including `.NET Runtime` (Windows only)
-  `.NET Runtime` only

Reply:
@publicarray  sorry, just cherry picked a commit of #4976 to fix the build.
I will now remove dotnet runtime from the PR...